webapp/src/ts/components/filters/analytics-filter/analytics-filter.component.ts

```diff
@@ -1,19 +1,54 @@
-import { AfterContentChecked, AfterContentInit, Component, Input, OnDestroy } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
-import { Subscription } from 'rxjs';
+import {
+  AfterContentChecked,
+  AfterContentInit,
+  Component,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output } from '@angular/core';
```

Comment:
```suggestion
  Output 
} from '@angular/core';
```
We usually close things in the next like, so it looks more like block